src/views/repository/Settings.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <repository-view-base
+    :repository="repository"
+    :loading="loading"
+    :error-code="errorCode">
+    <div
+      v-if="repository && repository.authorization"
```

Comment:
Check if user can write in this Repository, not just if user have authorization object.
Check if user is authenticated and after check if user can write.

Use `repository.authorization.can_write`.